Holovibes/sources/thread/camera_frame_read_worker.cc

```diff
@@ -59,18 +59,16 @@ void CameraFrameReadWorker::enqueue_loop(const camera::CapturedFramesDescriptor&
                                          const camera::FrameDescriptor& camera_fd)
 {
     cudaMemcpyKind copy_kind = captured_fd.on_gpu ? cudaMemcpyDeviceToDevice : cudaMemcpyHostToDevice;
-    for (unsigned i = 0; i < captured_fd.count1; ++i)
+    if (captured_fd.count1 > 0)
     {
-        auto ptr = (uint8_t*)(captured_fd.region1) + i * camera_fd.get_frame_size();
-        input_queue_.load()->enqueue(ptr, copy_kind);
+        auto ptr1 = (uint8_t*)(captured_fd.region1);
```

Comment:
Casting like in C is not clean use static_cast instead

Reply:
Done